serveradmin/backup.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+import zipfile
+
+from datetime import datetime
+
+from django.conf import settings
+
+
+from ideasbox import __version__
+
+
+def make_name():
+    """Return backup formatted file name."""
+    return '{}.zip'.format('_'.join([
+        settings.IDEASBOX_ID,
+        __version__,
+        datetime.now().strftime(Backup.DATE_FORMAT)
+    ]))
+
+
+class Backup(object):
+
+    FORMAT = 'zip'
+    DATE_FORMAT = "%Y%m%d%H%M"
```

Comment:
My concern is that I find `20150127223034` hardly readable. Something like `2015-01-27T22:30:34` looks more readable at first sight to me. Other option: just keep the iso date + counter `2015-01-27-001` for instance.